system_tests/full_challenge_impl_test.go

```diff
@@ -215,10 +222,12 @@ func setupSequencerInboxStub(ctx context.Context, t *testing.T, l1Info *Blockcha
 		big.NewInt(117964),
 		reader4844,
 		false,
+		espressoTEEVerifierAddr,
 	)
 	Require(t, err)
 	_, err = EnsureTxSucceeded(ctx, l1Client, tx)
 	Require(t, err)
+
```

Comment:
LGTM. Only that I think we should minimize our changes on nitro/celestia's files, which is good for syncing